mozmeao-fr/basket-prod/clock-deploy.yaml

```diff
@@ -197,11 +169,6 @@ spec:
           value: basket-prod
         - name: SUBHUB_OPP_RECORD_TYPE
           value: 0124O000000R0ODQA0
-        - name: SUPERTOKEN
```

Comment:
Thinking aloud: I see a ref to `supertoken` in `basket-client` https://github.com/mozilla/node-basket/blob/master/index.js#L61 -- but it doesn't look like it's deployed as part of main basket (which makes sense - it's a client, after all)

Reply:
Yeah it was removed awhile ago and returns a 404 now... https://github.com/mozmeao/basket/blob/master/basket/news/views.py#L715